org.adempiere.base/src/org/compiere/model/MTable.java

```diff
@@ -915,7 +915,7 @@ public int getWindowIDFromMenu() {
 				+ "INNER JOIN AD_Menu m ON (a.AD_Window_ID=m.AD_Window_ID AND m.IsActive='Y' AND m.Action='W') "
 				+ "WHERE a.IsActive='Y' AND b.IsActive='Y' AND b.AD_Table_ID=? ORDER BY b.TabLevel, a.AD_Window_ID", getAD_Table_ID());
 	}
-
+	

```

Comment:
It seems there's no changes being make to this class. You can use Replace With > Head Revision to remove this class from PR.

Reply:
This did not work as expected. Removed the whitespace change, but the class remains in the PR.

Reply:
No I did not, I do not see any remaining whitespace changes in the PR, there are identical lines as "removed" and "added". These were created in the diff, when I merged my branch with master because of a conflict in MTable

Reply:
ic, if Head Revision is not working, "Replace With" > "Branch, Tag, Reference" > "Remote Tracking" > "upstream/master" should did the trick.

Reply:
At the end `git checkout master -- [filename]` command in terminal and it worked, did not try your solution yet.